packages/contracts-ts/wagmi.config.ts

```diff
@@ -0,0 +1,365 @@
+import { ContractConfig, defineConfig, Plugin } from '@wagmi/cli'
+import { actions, react } from '@wagmi/cli/plugins'
+import glob from 'glob'
+import { readFileSync, writeFileSync } from 'fs'
+import type { Abi, AbiFunction, Address } from 'abitype'
+import { isDeepStrictEqual } from 'util'
+
+/**
+ * Predeployed contract addresses
+ * In future it would be nice to have a json file in contracts bedrock be generated as source of truth
+ * Keep this in sync with op-bindings/predeploys/addresses.go in meantime
+ */
+const predeployContracts = {
+  LegacyMessagePasser: {
+    address: '0x4200000000000000000000000000000000000000',
+    introduced: 'Legacy',
+    deprecated: true,
+    proxied: true,
```

Comment:
Do we use these additional values?

Reply:
Not yet I included them here to encourage the docs team to generate docs from this since the docs use them